frontend/src/components/account/Auth.tsx

```diff
@@ -91,14 +91,22 @@ export function PureAuthToken({
   onChangeToken,
   onCloseError,
 }: PureAuthTokenProps) {
+  const focusedRef = React.useCallback(node => {
+    if (node !== null) {
+      node.setAttribute('tabindex', '-1');
+      node.focus();
+    }
+  }, []);
+
   return (
     <Box>
       <ClusterDialog useCover disableEscapeKeyDown disableBackdropClick>
-        <DialogTitle id="responsive-dialog-title">{title}</DialogTitle>
+        <DialogTitle ref={focusedRef} id="responsive-dialog-title">
+          {title}
+        </DialogTitle>
         <DialogContent>
           <DialogContentText>Please paste your authentication token.</DialogContentText>
           <TextField
-            autoFocus
```

Comment:
Yeah, I'll look into it. There's some changes to the focus stuff I want to do based on recommendations on how-to-modal-a11y. Changing the focus to the first text/input box, rather than the title. Also making sure the default action works as before.

Reply:
I followed this order for modal focus: first input if there, otherwise description or title. So for this one, the input field gets focused.